### PR TITLE
retire old bot comments

### PR DESCRIPTION
**Pull request by crimsonfaith91**
Addressing issue https://github.com/kubernetes/test-infra/issues/3462
Related to PR https://github.com/kubernetes/test-infra/pull/2891

/assign @fejta

### CODE REVIEW

File: prow/config/jobs_test.go [hunk 1]
```diff
@@ -180,7 +180,8 @@ func TestCommentBodyMatches(t *testing.T) {
 		Presubmits: map[string][]Presubmit{
 			"org/repo": {
 				{
-					Name:      "gce",
+					Name: "gce",
+					//should we change this to: /test <some-test-name> ?
```
Comment (crimsonfaith91) [inline prow/config/jobs_test.go]: @fejta I remembered that you suggested to not use new bot command format for testing in this file. As the old format will be retired, i think we can now use the new format here.

Comment (fejta) [inline prow/config/jobs_test.go]: Sure

Comment (crimsonfaith91) [inline prow/config/jobs_test.go]: thanks, i will use the new format here and send out another commit soon.

**Comment by fejta** [comment]
/assign @cjwagner 

Cole/Jun, will you two please work together to figure out the best time/way to merge/deploy? Thanks!

**Comment by crimsonfaith91** [comment]
@fejta Definitely! I will work with @cjwagner to deploy the change. I updated `prow/config/jobs_test.go` to use new format. Thanks!

File: prow/Makefile [hunk 2]
```diff
@@ -15,7 +15,7 @@
 all: build test
 
 
-HOOK_VERSION       ?= 0.139
+HOOK_VERSION       ?= 0.140
```
Comment (crimsonfaith91) [inline prow/Makefile]: @cjwagner bumped hook for prow. Not sure it is the right way though.

I think we can have 2 commits for this PR as each commit indicates different changes.

**Pull request merged**